src/utils/update/company.profile.mapper.to.oversea.entity.ts

```diff
@@ -0,0 +1,35 @@
+import { ICompanyDetails } from "model/company.profile.model";
+import { Address } from "../../model/data.types.model";
+
```

Comment:
don't use any - pass in CompanyProfile. Also function name seems long & strange.

Following is full example (so you need to remove the fields not passed by node api sdk):

export const mapCompanyProfileToEntity = (cp : CompanyProfile): Entity => {
  const isOnReg = cp.isOnRegisterInCountryFormedIn ? yesNoResponse.Yes : yesNoResponse.No;
  const entity : Entity = {
    name: cp.companyName,
    registration_number: cp.companyNumber,
    law_governed: cp.foreignCompanyDetails?.governedBy,
    legal_form: cp.foreignCompanyDetails?.legalForm,
    incorporation_country: cp.jurisdiction,
    public_register_name: cp.foreignCompanyDetails?.originatingRegistry.name,
    public_register_jurisdiction: cp.foreignCompanyDetails?.originatingRegistry.country,
    email: "", // private data
    principal_address: {
      property_name_number: cp.registeredOfficeAddress?.premises,
      line_1: cp.registeredOfficeAddress?.addressLineOne,
      line_2: cp.registeredOfficeAddress?.addressLineTwo,
      town: cp.registeredOfficeAddress.locality,
      county: cp.registeredOfficeAddress?.region,
      country: cp.registeredOfficeAddress?.country,
      postcode: cp.registeredOfficeAddress?.postalCode
    },
    service_address: {
      property_name_number: cp.serviceAddress?.premises,
      line_1: cp.serviceAddress?.addressLineOne,
      line_2: cp.serviceAddress?.addressLineTwo,
      town: cp.serviceAddress?.locality,
      county: cp.serviceAddress?.region,
      country: cp.serviceAddress?.country,
      postcode: cp.serviceAddress?.postalCode,
    }//,
    // is_on_register_in_country_formed_in: isOnReg // breaks node
    // ,
    // is_service_address_same_as_principal_address: yesNoResponse.No,
  };
  return entity;
};